src/search-injection/content_script.js

```diff
@@ -0,0 +1,64 @@
+import * as constants from './constants'
+import * as utils from './utils'
+import { handleRender } from './dom'
+
+import { SEARCH_CONN_NAME, CMDS } from '../overview/constants'
+
+const cmdHandler = ({ cmd, ...payload }) => {
+    // cmd: (string) status of the search result returned
+    // payload: (object) contains doc and totalCount
+    // Listens in the search port for message
+
+    switch (cmd) {
+        case CMDS.RESULTS:
+            console.log(payload)
+            // Render only if there is atleast one result
+            if (payload.searchResult.docs.length) {
+                handleRender(payload.searchResult)
+            }
+            break
+        case CMDS.ERROR:
+            break
+        default:
+            console.error(`Unknown command ${cmd} with payload \n${payload}`)
+    }
+}
+
+const search = query => {
+    // query: (string) query to be search
```

Comment:
For code doc, I would advise getting familiar with jsdoc which is what we're using in the rest of the project. Not super urgent - comments are better than nothing! - but just to keep in mind for future.

Reply:
Oh okay, I will take a look at jsdoc.